File: GFG/Array/Day2/EqualSum.java
public class EqualSum{
    String equilibrium(int arr[]) {
        // code here
        int sum=0;
        
        for(int n:arr){
            sum+=n;
        }
        
        int leftsum=0;
        
        for(int i=0;i<arr.length;i++){
            sum-=arr[i];
            if(leftsum==sum){
                return "true";
            }
            leftsum+=arr[i];
            // sum-=arr[i];
        }
        
        return "false";
    }
}

File: GFG/Array/Day2/EqualSum.py
class Solution:
	def equilibrium(self, arr):
		# code here
		summ = 0

		for num in arr:
			summ += num

		leftSum = 0

		for num in arr:
			summ -= num
			if summ == leftSum:
				return "true"
			leftSum += num

		return "false"